### PR TITLE
Add buy to port feature and sell feature.

### PR DESCRIPTION
**Pull request by amchengcuenca**
Sell feature will re-render the particular rows it is selling from. If the qty of sell exceeds the amount currently held, the row will get removed.

### CODE REVIEW

File: client/components/SellButton.jsx [hunk 1]
```diff
@@ -1,19 +1,37 @@
...
 import Button from '@mui/material/Button';
 
-function SellButton({ user, ticker, sellQty }) {
+function SellButton({ user, ticker, sellQty, setUserPort, userPort }) {
+  const userPortAfterSell = (sym, quant) => {
+    const newPortValues = [];
+    for (let i = 0; i < userPort.length; i++) {
+      if (userPort[i].symbol === sym) {
+        const soldTickerVals = { ...userPort[i] };
+        if (soldTickerVals.qty - Number(quant) <= 0) {
```
Comment (BoopdiBop) [inline client/components/SellButton.jsx]: If the quantity is more than I have to sell, I still want to add this userPort[i] to my newPortValues.  However, I just won't be changing the quantity.   if I have 5 shares of AAPL and I want to sell 10, then I'll just be left with my 5 shares.

**Review by BoopdiBop — COMMENTED** [review]
Look at comment.

**Comment by amchengcuenca** [comment]
checks if potential sell amount is larger than current quantity amount, returns the ticker untouched. Subtracts amount if there is enough quantity being held. removes row if quantity held drops to 0 after sell.

**Review by BoopdiBop — APPROVED** [review]
Great!

**Pull request merged**